GSA/digitalgov.gov
# Footer: Add Linkedin to social list & remove email us

## Summary

This PR will add LinkedIn to the social media list and remove the email us


Before:
<img width="181" alt="Screenshot 2023-10-03 at 10 45 59 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/d349606d-cad5-4352-b399-918221b6057b">

After:
<img width="181" alt="Screenshot 2023-10-03 at 10 52 53 AM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/a1a8fc16-46a4-495e-a498-6c6853004024">

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-add-linkedin/)



### Solution

### How To Test

1. pull down branch 
2. run `npm run start`
3. visit `localhost:1313` and scroll to the bottom and verify that the linkedin icon is present and takes you to the digital.gov linkedin page and that the email us has been removed (see below for full requirements)

- The Social media list needs a new item
- LinkedIn icon
- that links to: https://www.linkedin.com/company/digitalgov-gsa/

---

### Dev Checklist
